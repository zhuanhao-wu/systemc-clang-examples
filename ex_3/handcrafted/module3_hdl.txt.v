module top_0(
input wire clk,
input wire integer int_port 
);
integer var;
integer x;
integer y;


endmodule // top_0