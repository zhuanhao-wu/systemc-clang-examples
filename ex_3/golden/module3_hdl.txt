  [
    (hModule top_0)
    [
      (hPortin clk)
      [
        (hType sc_in)
      ]
      (hPortin int_port)
      [
        (hType sc_in)
        (hType int)
      ]
      (hVardecl var)
      [
        (hType int)
      ]
      (hVardecl x)
      [
        (hType int)
      ]
      (hVardecl y)
      [
        (hType int)
      ]
    ]
    [
    ]
  ]
